python/tskit/intervals.py

```diff
@@ -1,6 +1,6 @@
 # MIT License
 #
-# Copyright (c) 2023 Tskit Developers
+# Copyright (c) 2024 Tskit Developers
```

Comment:
Not sure why this file got touched?

Reply:
It didn't, but one of the CI checks for some reason was complaining about in in a previous PR. Happy to remove

Reply:
I just removed the commit that did the year updates for extra untouched files that were previously complained about.